ming_sim/action_materialize.py

```
@@ -2027,13 +2281,100 @@ def _materialize_appointment(ctx: MaterializeCtx) -> None:
+            pend_for_minister=ctx.pend_for_minister,
+            content=content_ref,
+        )
+        if status in {"ambiguous", "含糊"}:
+            ctx.out["directive_confirmation_ambiguous"] = _office_path_ambiguous_payload(
+                session.db,
+                int(session.state.turn),
+                pend_for_minister=ctx.pend_for_minister,
+            )
+            return
+        if status == "hit" and row is not None:
+            # 同人同职再发任命+路径：no-op 去重，中旨/任别并入既有条
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)